db/migrate/20150316144315_remove_name_from_course_users.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveNameFromCourseUsers < ActiveRecord::Migration
+  def change
+    remove_column :course_users, :name
```

Comment:
I think this field might be essential, because we want do course specific name in v2, Thus teachers can prevent student from change their names in course. I've a email about this, will forward you.